csharp/extractor/Semmle.Extraction.CSharp/Analyser.cs

```diff
@@ -456,7 +456,7 @@ public bool LogRoslynArgs(string[] roslynArgs, string extractorVersion)
             bool argsWritten;
             using (var streamWriter = new StreamWriter(new FileStream(tempFile, FileMode.Append, FileAccess.Write)))
             {
-                streamWriter.WriteLine($"# Arguments to Roslyn: {string.Join(' ', roslynArgs)}");
+                streamWriter.WriteLine($"# Arguments to Roslyn: {string.Join(' ', roslynArgs.Where(arg => arg[0] != '@'))}");
```

Comment:
`!arg.StartsWith("@")`

and fix line 19 as well?

It's just possible that `arg` is empty.

Reply:
Yeah, that is better.